python/python/bystro/utils/compress.py

```diff
@@ -31,24 +31,61 @@ def _get_gzip_program_path() -> str:
 IS_BGZIP = GZIP_EXECUTABLE.endswith("bgzip")
 
 
-def get_compress_from_pipe_cmd(out: str) -> str:
+def get_compress_from_pipe_cmd(
```

Comment:
also updated this method to support piped commands, to be symmetrical with the decompress method